### DISCUSSION TITLE
Suggestion - Official Matrix Space/Room

### DISCUSSION THREAD

**Discussion by dreaddr** [linuxdeepin/developer-center#2842] [Features Request & Ideas | 特性请求 & 头脑风暴]
Please consider making an official [matrix] ([matrix.org](https://matrix.org)) space or room for the community to engage in, and bridge it to the #IRC channel (matrix.org/bridges/#irc) as well so that conversations aren't lost.

Yet another way Deepin can engage with its users more directly for support or otherwise. Maybe host it on Deepin officially through direct hosting (Perhaps chat.deepin.org or matrix.deepin.org, inspired by chat.mozilla.org)

**Comment by peeweep** [comment]
I prefer to use [chat.mozilla.org](https://chat.mozilla.org/), can we create a room on it and use the matrix bridge to link the telegram group?

Matrix-TG Bridge:
* https://matrix.org/docs/projects/bridge/mautrix-telegram (registration appservice maybe required selfhost homeserver)
* https://matrix.org/docs/projects/bot/matterbridge

**Comment by justforlxz** [comment]
Good news. Now deepin has created the matrix and is in sync with telegram(@deepin) and irc(#deepin). https://matrix.to/#/#deepin:matrix.org

<img width="1307" alt="image" src="https://user-images.githubusercontent.com/12298476/182085182-f16006fa-a23a-4723-955d-ac78239bfcb0.png">

<img width="882" alt="image" src="https://user-images.githubusercontent.com/12298476/182085311-e0452549-d78d-4fcf-8a36-a0f0bcf768b9.png">
